### ISSUE TITLE
Failure to convert variable to continuous: values contain text[Bug]: 

### ISSUE THREAD

**Issue by cudzz** [jasp-stats/jasp-issues#2626]
### JASP Version

0.18.3

### JASP Module

Unrelated

### What analysis are you seeing the problem on?

Before-- Just opening data

### What OS are you seeing the problem on?

macOS Intel

### Bug Description

have data that I uploaded to JASP before (with fewer subjects) where the program automatically interpreted the variables I needed to be continuous as such. Now after cleaning up my fresh subset of data from new subjects in excel and exporting it as a cvs file, the variables will not change. They also won't change manually when I try to and the following message pops up: "failure to convert variable to continuous: values contain text".

I have also checked my data multiple times and don't have text in the columns that I want to change.

### Expected Behaviour

It should look like this:
<img width="1135" alt="Screen Shot 2024-03-04 at 11 28 39" src="https://github.com/jasp-stats/jasp-issues/assets/162157359/e8847ebf-58ac-47b0-baec-ef8797f78c7e">

It's showing up like this: and I cannot change it.
<img width="1135" alt="Screen Shot 2024-03-04 at 11 29 10" src="https://github.com/jasp-stats/jasp-issues/assets/162157359/e788acfa-9b0e-4a67-8e17-2c4f73a48d25">

### Steps to Reproduce

No idea.

### Log (if any)

[For JASP-Table 1.csv.zip](https://github.com/jasp-stats/jasp-issues/files/14511197/For.JASP-Table.1.csv.zip)

CVS file used is linked.

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [ ] I have accurately described the bug, and steps to reproduce it.

**Comment by shun2wang** [comment]
Yes exactly, if I look into the csv file it seems has about 100 empty rows at bottom, if I remove them then worked. in .csv we can see something like:

```
"
","
","
","
","
","
","
","
","
","
","
","
","
```

@boutinb it's really tricky, I remember we discussed we may find a way to trim empty rows,for now, we can at least detect active rows (black row numbers). We can work backwards from the last active row to determine whether it is a empty row, and then select the longest column as whole active rows number. For example:

| JASP row No. | col1 | col2 | col3 | col4 |
|-----|------|------|------|------|
| 1   | a    | 21   | w    | q    |
| 2   | a    | 32   | q    |      |
| 3   |      | 32   | a    |      |
| 4   |      |      | x    |      |
| 5   |      |      |     |      |

so we should trim row 5.  blank rows at the top has little impact because it is easy for users to spot it and decide whether to remove it?Or only for this issue, we can consider how to skip empy values when convert the variable?

**Comment by cudzz** [comment]
Oh thank you. I got rid of the 11 rows and it worked! Somewhere along the procedure i must have copied too much over.
